web-ts/src/routes/OffersPage.tsx

```diff
@@ -11,25 +28,85 @@ type PropsType = {
-  const { languageCode } = match.params
+const OffersPage = ({ cityModel, match, location, languages }: PropsType): ReactElement => {
+  const { languageCode, cityCode } = match.params
+  const pathname = normalizePath(location.pathname)
+  const { t } = useTranslation('offers')
+  const viewportSmall = false
 
   const toolbar = (openFeedback: (rating: FeedbackRatingType) => void) => (
-    <LocationToolbar openFeedbackModal={openFeedback} viewportSmall={false} />
+    <LocationToolbar openFeedbackModal={openFeedback} viewportSmall={viewportSmall} />
   )
 
+  const requestOffers = useCallback(async () => {
+    return createOffersEndpoint(cmsApiBaseUrl).request({ city: cityCode, language: languageCode })
+  }, [cityCode, languageCode])
+  const { data: offers, loading, error: offersError } = useLoadFromEndpoint(requestOffers)
+
+  const toTileModels = useCallback(
+    (offers: Array<OfferModel>): Array<TileModel> => {
+      return offers.map(offer => {
+        let path = offer.path
+
+        if (offer.alias === SPRUNGBRETT_OFFER) {
+          path = createPath(SPRUNGBRETT_OFFER_ROUTE, { cityCode, languageCode })
+        }
+
+        return new TileModel({
+          title: offer.title,
+          // the url stored in the sprungbrett offer is the url of the endpoint
+          path: path,
+          thumbnail: offer.thumbnail,
+          // every offer except from the sprungbrett offer is just a link to an external site
+          isExternalUrl: path === offer.path,
+          postData: offer.postData
+        })
+      })
+    },
+    [cityCode, languageCode]
+  )
+
+  const languageChangePaths = languages.map(({ code, name }) => {
+    const offersPath = createPath(OFFERS_ROUTE, { cityCode, languageCode: code })
+    return {
+      path: offersPath,
+      name,
+      code
+    }
+  })
+
+  const locationLayoutParams = {
+    cityModel,
+    viewportSmall,
+    feedbackTargetInformation: { path: createPath(OFFERS_ROUTE, { cityCode, languageCode }) },
+    languageChangePaths,
+    route: OFFERS_ROUTE,
+    languageCode,
+    pathname,
+    toolbar
+  }
+
+  if (loading) {
+    return (
+      <LocationLayout isLoading {...locationLayoutParams}>
+        <LoadingSpinner />
+      </LocationLayout>
+    )
+  }
+
+  if (!offers) {
+    const error = offersError || new Error('Offers should be available.')
+
+    return (
+      <LocationLayout isLoading={false} {...locationLayoutParams}>
+        <FailureSwitcher error={error} />
+      </LocationLayout>
+    )
+  }
+
   return (
-    <LocationLayout
-      cityModel={cityModel}
-      toolbar={toolbar}
-      viewportSmall={false}
-      feedbackTargetInformation={null}
-      languageChangePaths={null}
-      isLoading={false}
-      route={OFFERS_ROUTE}
-      languageCode={languageCode}
-      pathname={location.pathname}>
-      <div>OffersPage</div>
+    <LocationLayout isLoading={false} {...locationLayoutParams}>
+      <Tiles title={t('offers')} tiles={toTileModels(offers)} />
```

Comment:
Am I missing something? Somehow the translation does not work here -.-

Reply:
Had the same problem already and was very puzzled. You are importing `useTranslation` from the mock instead of `react-i18next` :)

Reply:
We could exclude mocks from IntelliJ auto import:
Settings > Code Style > Typescript > Imports > Do not import exactly from